cmd/snap-seccomp/main_test.go

```diff
@@ -190,6 +191,7 @@ func (s *snapSeccompSuite) SetUpSuite(c *C) {
 		}
 		if output, err := cmd.CombinedOutput(); err != nil {
 			fmt.Printf("cannot build multi-lib syscall runner: %v\n%s", err, output)
+			s.cannotCheckCompatArch = true
```

Comment:
I wonder if it would be better to check to see if the gnu/stubs-32.h exists and if not (perhaps similar to how autotools will build something to decide), then set cannonCheckCompatArch so as not to mask compilation failures we want to address.

Reply:
Yeah, I think it makes sense to add some safeguard here to not loose the test if for some reason the compiler changes and our test stops compiling on ubuntu 64bit for unrelated reason.

Reply:
Thanks for suggestions. I've updated the patch. Now it will check for existence of `/usr/include/gnu/stubs-32.h`. I've checked that this is the location for Ubuntu 14.04, 16.04, Fedora and Arch.

Reply:
Also, force pushed, trying to keep this as a single patch.